changelog.d/9238.misc

```diff
@@ -0,0 +1 @@
+Add ratelimited to 3PID `/requestToken` API.
```

Comment:
I think this got pulled over from the wrong branch.

Reply:
ETOOMANYBRANCHES

Reply:
How did that event happen :/